baseapp-profiles/baseapp_profiles/models.py

```diff
@@ -135,6 +133,14 @@ def get_graphql_object_type(cls):
 
         return ProfileObjectType
 
+    @property
+    def name(self):
+        if hasattr(self, "user") and self.user:
+            return f"{self.user.first_name} {self.user.last_name}"
+        if hasattr(self, "organization") and self.organization:
+            return self.organization.name
```

Comment:
this won't work since `name`  doesn't exist under `Organization`

Reply:
I introduced a name field in the org model